src/test/java/net/masterthought/cucumber/ChartUtilTest.java

```diff
@@ -0,0 +1,45 @@
+package net.masterthought.cucumber;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import net.masterthought.cucumber.json.support.TagObject;
+import net.masterthought.cucumber.util.ChartUtil;
+
+public class ChartUtilTest {
+
+    @Test
+    public void testGetTags() throws Exception {
+        List<TagObject> tagObjectList = new ArrayList<TagObject>();
+        TagObject tag = new TagObject("TestTagName");
+        tagObjectList.add(tag);
+        String result = ChartUtil.getTags(tagObjectList);
+        Assert.assertEquals("['TestTagName']", result);
+    }
+
+    @Test
+    public void testGetTags_withoutTags() throws Exception {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Method names should not contain underscores](https://www.codacy.com/app/damianszczepanik/cucumber-reporting/file/1747396588/issues/source?bid=3059308&fileBranchId=3060128#l24)